index.d.ts

```diff
@@ -80,10 +69,24 @@ declare function build(
  * @returns {Transform} the wrapped split2 instance
  */
 declare function build(
-  fn: (transform: Transform & OnUnknown) => Transform & OnUnknown,
+  fn: (transform: Transform & build.OnUnknown) => Transform & build.OnUnknown,
   opts: EnablePipelining
 ): Transform;
 
-export { OnUnknown };
+declare namespace build {
```

Comment:
Declaring a namespace and merging the interface onto it is what prevents this from making a backwards breaking change, i.e. `import build, { OnUnknown } from "pino-abstract-transport"` still works.